projects/WeTek_Core/initramfs/platform_init

```diff
@@ -1,20 +1,20 @@
 #!/bin/sh
 ################################################################################
-#      This file is part of OpenELEC - http://www.openelec.tv
-#      Copyright (C) 2014 Alex Deryskyba (alex@codesnake.com)
```

Comment:
sorry, I missed the old PR. copyright of this code may be LibreELEC. copyright != authorship. you can't just replace the original author of this code with yourself. you are not the author of this code.